extensions/wikidata/src/org/openrefine/wikidata/schema/WbDateConstant.java

```diff
@@ -104,9 +104,15 @@ public static TimeValue parse(String datestamp)
         Date bestDate = null;
         int precision = 0; // default precision (will be overridden if successfully parsed)
         int maxLength = 0; // the maximum length parsed
+        boolean bceFlag = false; // judge whether this is a BCE year
         String calendarIri = TimeValue.CM_GREGORIAN_PRO; // Gregorian calendar is assumed by default
         
         String trimmedDatestamp = datestamp.trim();
+
+        if(trimmedDatestamp.startsWith("-")){
+            trimmedDatestamp = trimmedDatestamp.replace("-", "");
```

Comment:
Replacing every occurrence of `-` in the string is likely to cause problems if we want greater precision than years.
For instance, it seems that we would not be able to parse `-1234-10` (October 1234 BCE). Perhaps you could add a test case for this and use another method to remove the leading minus sign?